### PR TITLE
feat(panel, shell-panel): adds width-scale default to panel. adds width-scale property and styles to shell-panel

### PR DESCRIPTION
**Pull request by asangma**
**Related Issue:** #1107

## Summary

- Panel: adds default width-scale
- ShellPanel: adds width-scale and associated styles

cc @kevindoshier @AdelheidF

### CODE REVIEW

**Review by driskull — APPROVED** [review]
👍

File: src/demos/shell/demo-app-advanced-2.html [hunk 1]
```diff
@@ -131,7 +131,7 @@
               </calcite-action-group>
             </calcite-action-bar>
               <calcite-panel
-                width-scale="m"
+
```
Comment (jcfranco) [inline src/demos/shell/demo-app-advanced-2.html:134]: Nitpick: can you remove this empty newline?

File: src/components/calcite-panel/calcite-panel.tsx [hunk 2]
```diff
@@ -85,9 +85,9 @@ export class CalcitePanel {
   @Prop({ reflect: true }) heightScale: CalciteScale;
 
   /**
-   * This sets width and max-width of the content area.
+   * This sets width and max-width of the panel.
```
Comment (jcfranco) [inline src/components/calcite-panel/calcite-panel.tsx:88]: Nitpick: does this just state that it sets the width? I think `max-width` is more of an implementation detail.

**Review by jcfranco — APPROVED** [review]
👍

File: src/components/calcite-panel/calcite-panel.tsx (hunk 2 above)
Comment (asangma) [reply src/components/calcite-panel/calcite-panel.tsx:88]: Copy that.

File: src/demos/shell/demo-app-advanced-2.html (hunk 1 above)
Comment (asangma) [reply src/demos/shell/demo-app-advanced-2.html:134]: Will do.

**Pull request merged**